### DISCUSSION TITLE
Internet connection with GSM module.

### DISCUSSION THREAD

**Discussion by urielc30** [community/community#41541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I am doing a project of sending coordinates with a Raspberry pi 4 and I want to use an internet connection from a module (I use the sim7000g module). When doing the configuration through AT command this assigns me an IP value 0.0.0.0 and it is impossible to send my data through the MQTT protocol. if you have any advice it would be very helpful.

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @urielc30! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
